Formula/csound.rb

```diff
@@ -43,13 +43,20 @@ class Csound < Formula
   depends_on "portmidi"
   depends_on "python@3.9"
   depends_on "stk"
-  depends_on "wiiuse"
 
   uses_from_macos "bison" => :build
   uses_from_macos "flex" => :build
   uses_from_macos "curl"
   uses_from_macos "zlib"
 
+  on_macos do
+    depends_on "wiiuse"
+  end
+
+  on_linux do
+    depends_on "gcc" => :build
```

Comment:
```suggestion
    depends_on "gcc"
```

Reply:
You also need to add `fails_with gcc: "5"` after this.  With those two changes this PR works when I test it locally in our Docker image.